### DISCUSSION TITLE
Still paraistic manager doesn't open after reboot

### DISCUSSION THREAD

**Discussion by elswerky** [mywalkb/LSPosed_mod#109] [General]
Bro 
As told you before,still parasitic manager doesn't open after reboot on tab S7 plus android 13 intuitive 5.1 latest software binary 5 

And by this some modules don't function as intended 

I tested action releases till #105

](
![Screenshot_20240601_045035_Shell](https://github.com/mywalkb/LSPosed_mod/assets/26687459/be29f7a7-172d-4db4-af28-fad2c1c83f30)

**Comment by elswerky** [comment]
@mywalkb bro , I downgraded super partition of tab S7 plus to an older binary update via twrp and found lspsoed has no issue and xposed edge module works fine 


And this confirm what I said before about last few updates of Samsung oneui in ineui 5,1 ,Samsung made changes which makes lsposed not function well 
I see this is a major issue if you can try to fix 

And I told you before about it many times

**Comment by mywalkb** [comment]
Post your logs, so I can try to understand which is the issue.
Have you tried to install manager?

**Reply by elswerky** [reply]
Bro 
I found that this issue occurs only with OneUi 5.1.1 android 13 
Doesn't occur with oneui 5.1 
Not tested on android 14 , Oneyi 6


So I think you should test it on any samsung device running oneui 5.1.1 and you may be able to fix it as I don't know how to take logs  and not experienced in this 


Just testbon any device with Oneui 5.1.1

**Comment by Luisa-Rosas** [comment]
Hello, I have the exact same problem on a motorola edge 30 android 13...
